Makefile

```diff
@@ -1,13 +1,21 @@
-BUILD_NUMBER := 1
+# Bump this on subsequent build, reset on new version or public release.
+BUILD_NUMBER := 35
+
 CP_VERSION := 3.3.0-SNAPSHOT
 
 COMPONENTS := base zookeeper kafka kafka-rest schema-registry kafka-connect-base kafka-connect enterprise-control-center kafkacat enterprise-replicator enterprise-kafka kafka-streams-examples
 COMMIT_ID := $(shell git rev-parse --short HEAD)
 MYSQL_DRIVER_VERSION := 5.1.39
 
-CONFLUENT_DEB_REPO := http://packages.confluent.io
-CONFLUENT_RPM_REPO := http://packages.confluent.io
-APT_ALLOW_UNAUTHENTICATED := false
+# CONFLUENT_DEB_REPO := http://packages.confluent.io
+CONFLUENT_DEB_REPO := https://s3-us-west-2.amazonaws.com/jenkins-confluent-packages/packaging-3.3.x/35
```

Comment:
You can currently pass them into make via command line args or a local.make file.